sonar-ws/src/main/gen-java/org/sonarqube/ws/QualityProfiles.java

```diff
@@ -0,0 +1,2483 @@
+            java.lang.String value) {
+          if (value == null) {
+    throw new NullPointerException();
+  }
+  bitField0_ |= 0x00000040;
+          parentName_ = value;
+          onChanged();
+          return this;
+        }
+        /**
+         * <code>optional string parentName = 7;</code>
+         */
+        public Builder clearParentName() {
+          bitField0_ = (bitField0_ & ~0x00000040);
+          parentName_ = getDefaultInstance().getParentName();
+          onChanged();
+          return this;
+        }
+        /**
+         * <code>optional string parentName = 7;</code>
+         */
+        public Builder setParentNameBytes(
+            com.google.protobuf.ByteString value) {
+          if (value == null) {
+    throw new NullPointerException();
+  }
+  bitField0_ |= 0x00000040;
+          parentName_ = value;
+          onChanged();
+          return this;
+        }
+
+        private boolean isDefault_ ;
+        /**
+         * <code>optional bool isDefault = 8;</code>
+         */
+        public boolean hasIsDefault() {
+          return ((bitField0_ & 0x00000080) == 0x00000080);
+        }
+        /**
+         * <code>optional bool isDefault = 8;</code>
+         */
+        public boolean getIsDefault() {
+          return isDefault_;
+        }
+        /**
+         * <code>optional bool isDefault = 8;</code>
+         */
+        public Builder setIsDefault(boolean value) {
+          bitField0_ |= 0x00000080;
+          isDefault_ = value;
+          onChanged();
+          return this;
+        }
+        /**
+         * <code>optional bool isDefault = 8;</code>
+         */
+        public Builder clearIsDefault() {
+          bitField0_ = (bitField0_ & ~0x00000080);
+          isDefault_ = false;
+          onChanged();
+          return this;
+        }
+
+        private long activeRuleCount_ ;
+        /**
+         * <code>optional int64 activeRuleCount = 9;</code>
+         */
+        public boolean hasActiveRuleCount() {
+          return ((bitField0_ & 0x00000100) == 0x00000100);
+        }
+        /**
+         * <code>optional int64 activeRuleCount = 9;</code>
+         */
+        public long getActiveRuleCount() {
+          return activeRuleCount_;
+        }
+        /**
+         * <code>optional int64 activeRuleCount = 9;</code>
+         */
+        public Builder setActiveRuleCount(long value) {
+          bitField0_ |= 0x00000100;
+          activeRuleCount_ = value;
+          onChanged();
+          return this;
+        }
+        /**
+         * <code>optional int64 activeRuleCount = 9;</code>
+         */
+        public Builder clearActiveRuleCount() {
+          bitField0_ = (bitField0_ & ~0x00000100);
+          activeRuleCount_ = 0L;
+          onChanged();
+          return this;
+        }
+
+        private long projectCount_ ;
+        /**
+         * <code>optional int64 projectCount = 10;</code>
+         */
+        public boolean hasProjectCount() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)